### PR TITLE
Switch to Survey Gizmo for feedback

### LINKED ISSUE #568: Implement new SurveyGizmo-based experiment feedback flow
Here's the UX https://people.mozilla.org/~jgruen/test-pilot/activity-stream.html (enable the experiment then click the feedback link)

Here's the state diagram: https://www.lucidchart.com/documents/view/bc257f5b-d2f9-4e23-8c60-fdb91b274bc6

### PR DESCRIPTION
**Pull request by meandavejustice**
## BLOCKED by #623

This pr is dependent on #623 being merged first.

~~Waiting on Survey Gizmo URLs. Opening for discussion / early review.~~
- update copy and remove outdated feedback refs in `locales/en-US/app.l20n`
- remove outdated feedback handling from `app/views/experiment-page.js`
- remove outdated feeback form styling from `styles/_modals.scss`
- update & rename `test/views/disable-dialog-view.js` to remove outdated feedback forms
- rename `views/feedback-view.js` to `views/disable-dialog-view.js`
- fixes #568

cc/ @johngruen for review

### CODE REVIEW

File: testpilot/frontend/static-src/app/views/experiment-page.js [hunk 1]
```diff
@@ -252,12 +253,7 @@ export default PageView.extend({
       id: 'disabled-feedback',
       experiment: this.model.url,
       title: 'feedbackUninstallTitle',
-      questions: [
-        { value: 'broken', title: 'feedbackUninstallAnswerBroken' },
-        { value: 'dislike', title: 'feedbackUninstallAnswerDislike' },
-        { value: 'notuseful', title: 'feedbackUninstallAnswerNotUseful' },
-        { value: 'other', title: 'feedbackUninstallAnswerOther' }
-      ],
+      surveyUrl: `http://meandave.com/faux_survey/?ref=disable&experiment=${this.model.title}`,
```
Comment (pdehaan) [inline testpilot/frontend/static-src/app/views/experiment-page.js]: Is the model `title` url-encoded?

Comment (meandavejustice) [inline testpilot/frontend/static-src/app/views/experiment-page.js]: oh good thinking @pdehaan

File: testpilot/frontend/static-src/app/views/feedback-view.js [hunk 2]
```diff
@@ -7,78 +6,39 @@ export default BaseView.extend({
...
           </div>
+          <p class="centered" data-lion-id="feedbackUninstallCopy">
+            Your participation in Firefox Test Pilot means a lot!
+            Please check out our other experiments, and stay tuned for more to come!
+          </p>
+
           <div class="modal-actions">
-            <button data-hook="submit-feedback" data-l10n-id="feedbackSubmitButton" class="submit button default large">Submit Feedback</button>
-            <a data-l10n-id="feedbackCancelButton" class="cancel" href="">Cancel</a>
+            <a data-lion-id="feedbackSubmitButton" href="{{surveyUrl}}" class="button default large quit">Take a quick survey</a>
```
Comment (pdehaan) [inline testpilot/frontend/static-src/app/views/feedback-view.js]: Data-lion-id? :cat:

File: testpilot/frontend/static-src/app/views/feedback-view.js [hunk 3]
```diff
@@ -7,78 +6,39 @@ export default BaseView.extend({
...
-            <div class="additional-feedback">
-              <p data-l10n-id="feedbackExtraCaption" class="text-area-label">Do you have any additional feedback? (optional)</p>
-              <textarea data-l10n-id="feedbackExtraTextarea"
-                placeholder="what's on your mind..."></textarea>
-            </div>
+        <div class="modal-content">
+          <div class="copter-wrapper fly-down">
+            <div class="copter"></div>
           </div>
+          <p class="centered" data-lion-id="feedbackUninstallCopy">
```
Comment (pdehaan) [inline testpilot/frontend/static-src/app/views/feedback-view.js]: Lion -> l10n?

Comment (meandavejustice) [inline testpilot/frontend/static-src/app/views/feedback-view.js]: lol my autocomplete must have goofed.

Comment (pdehaan) [inline testpilot/frontend/static-src/app/views/feedback-view.js]: autocomplete? You using MS Word to write JavaScript?

untitled

Comment (chuckharmston) [inline testpilot/frontend/static-src/app/views/feedback-view.js]: With `evil-mode`, I presume.

Comment (meandavejustice) [inline testpilot/frontend/static-src/app/views/feedback-view.js]: nope, `broken pinky/remapped caps lock` mode. Like RMS hisself

**Comment by lmorchard** [comment]
Unassigning myself, because I'm not (yet) actively working on this

File: testpilot/frontend/static-src/app/views/experiment-page.js [hunk 4]
```diff
@@ -243,21 +244,20 @@ export default PageView.extend({
...
       id: 'disabled-feedback',
       experiment: this.model.url,
       title: 'feedbackUninstallTitle',
-      questions: [
-        { value: 'broken', title: 'feedbackUninstallAnswerBroken' },
-        { value: 'dislike', title: 'feedbackUninstallAnswerDislike' },
-        { value: 'notuseful', title: 'feedbackUninstallAnswerNotUseful' },
-        { value: 'other', title: 'feedbackUninstallAnswerOther' }
-      ],
+      surveyUrl: `http://meandave.com/faux_survey/?${queryParams}`,
```
Comment (lmorchard) [inline testpilot/frontend/static-src/app/views/experiment-page.js]: I could be wrong, but my impression is that the SurveyGizmo URLs are going to be altogether different for each experiment. Assuming that's the case, they'll need to be supplied in new fields on the Experiment model, served up in the API results, and pulled in dynamically. The query parameters are needed to grab additional user feedback like the star ratings, but beyond that I think each experiment gets its own set of surveys.

File: testpilot/frontend/static-src/app/views/feedback-view.js [hunk 5]
```diff
@@ -1,84 +1,45 @@
-import app from 'ampersand-app';
 import BaseView from './base-view';
```
Comment (lmorchard) [inline testpilot/frontend/static-src/app/views/feedback-view.js]: Do we still need this feedback subview? I think the feedback button is just supposed to open the SurveyMonkey URL in a new tab without a modal dialog. Was planning to delete this subview altogether as part of #567

Comment (meandavejustice) [inline testpilot/frontend/static-src/app/views/feedback-view.js]: This is being used for the Disable dialog. I suppose I could rename it if it is only used there now.

Comment (lmorchard) [inline testpilot/frontend/static-src/app/views/feedback-view.js]: Ah yeah, okay... I missed that part of the prototype. Renaming the view to reflect it just being used for experiment disable might make things clearer

**Comment by lmorchard** [comment]
<strike>FWIW I think you can toss a `fixes #567` into this, too, because this PR effectively covers the old disable / feedback survey dialogs</strike>

Oh, wait, no: The backend feedback models & API still need to be removed, so I still have work to do in #567

**Comment by lmorchard** [comment]
This needs a rebase since #623 was merged. But, I think it'll be a quick r+ & merge from me, after that.

**Comment by lmorchard** [comment]
I went ahead and rebased this as #638, since it looks like most of the conflicts were from my onboarding stuff.

**Pull request closed without merging**